nightwatch/features/event-reconciliation/reconciled.feature

```diff
@@ -0,0 +1,151 @@
+Feature: Reconciled
+
+  Background:
+    Given I am a logged in user
+    And There are no existing centres
+    And The following centres exist:
+      | name | male_capacity | female_capacity | male_cid_name  | female_cid_name |
+      | one  | 1000          | 10000           | oneman,onebman | onewoman        |
+    Given I am on the wallboard
+
+  Scenario: Reconciled Check In Event and Movement In does not show as Unexpected In and Scheduled In or affect Availability
+    Given The Centre "one" should show the following under "Male":
+      | Contractual Capacity   | 1000 |
```

Comment:
conversation: do we need to assert everytime that the centre has been created cleanly?